README.md

```diff
@@ -61,6 +61,10 @@ module "aws_vault" {
 | elb_route53_public_dns_name_vault | The Route53 name attached to the Vault ELB, if spcified in variables. |
 | asg_name_consul_cluster | Autoscaling group name of the Consul cluster. |
 | asg_name_vault_cluster | Autoscaling group name of the Vault cluster. |
+| consul_instance_ids | List of EC2 instance ids of deployed consul hosts |
```

Comment:
```bash
$ terraform apply
var.allowed_account_ids
  The AWS account id to which this project is strictly bound.

  Enter a value: 608780100653

var.bastion_ingress_cidr_ssh
  List of CIDR's from which you are allowed to ssh into the bastion host.

  Enter a value: ^C
Interrupt received.
Please wait for Terraform to exit or data loss may occur.
Gracefully shutting down...

Error: Error asking for user input: Error asking for bastion_ingress_cidr_ssh: interrupted
```
I am not able to easily test this PR, because the example is not self-contained. Meaning, I have to type a use case for the example to work. Please accompany your examples by var files

Reply:
@danvaida 
This has been fixed separately as it was/is not part of this PR. See here: https://github.com/Flaconi/terraform-aws-vault/pull/5